### PR TITLE
Update Config to reflect documentation and update docs to be clearer

### PR DESCRIPTION
**Pull request by thorerik**
Config didn't match what documentation said it should be, updated this to default safe to true.

Additionally we found the docs around safe to be a little ambiguous, so I've attempted to make it less so.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
The default needs to be false to be consistent with commonmark please.

File: config/markdown.php [hunk 1]
```diff
@@ -107,10 +107,10 @@
     |
     | This option specifies if raw HTML should be allowed in the document.
```
Comment (GrahamCampbell) [inline config/markdown.php]: Please match this up with the description in the readme like it was already.

**Comment by thorerik** [comment]
Updated and squashed the commits :+1:

**Comment by GrahamCampbell** [comment]
Thank you. :)

**Pull request merged**